File: Python3/2278_percentage_of_letter_in_string.py
# https://leetcode.cn/problems/percentage-of-letter-in-string/

class Solution:
    def percentageLetter(self, s: str, letter: str) -> int:
        n, cnt = len(s), 0
        for ch in s:
            cnt += ch == letter
        return int(cnt / n * 100)

File: Cpp/2278_percentage_of_letter_in_string.cpp
// https://leetcode.cn/problems/percentage-of-letter-in-string/

#include <iostream>
#include <string>

using namespace std;

class Solution {
public:
    int percentageLetter(string s, char letter) {
        int n = s.size(), cnt = 0;
        for (char &ch: s) {
            cnt += ch == letter;
        }
        return cnt * 100 / n;
    }
};

int main() {
    Solution sol;
    string s = "foobar";
    char letter = 'o';
    cout << sol.percentageLetter(s, letter) << '\n';
    return 0;
}